README.md

```diff
@@ -304,9 +304,9 @@ apollo {
 // With:
 apollo {
   onCompilationUnit {
-     schemaFile = "/path/to/your/schema.json"
+     schemaFile.set("/path/to/your/schema.json")
```

Comment:
Shouldn't this be
 ```
schemaFile = file("/path/to/your/schema.json")
```

Reply:
Yup, my bad, I wanted to make something that worked in kotlin and groovy but forgot the `file` part...

Reply:
I think `schemaFile.set(file("/path/to/your/schema.json"))` would work equally well in Groovy and Kotlin.